hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/volume/acl/TestOMVolumeAddAclRequest.java

```diff
@@ -91,7 +91,6 @@ public void testValidateAndUpdateCacheSuccess() throws Exception {
 
     OmOzoneAclMap aclMapAfterSet = omMetadataManager
         .getVolumeTable().get(volumeKey).getAclMap();
-    Assert.assertNotEquals(aclMapBeforeSet, aclMapAfterSet);
```

Comment:
Why is this assert removed?

Reply:
Because it is getting from the cache, it will have same reference.